### PR TITLE
add features as cols for show vendors

### PR DESCRIPTION
**Pull request by charles-tan**
1. adds features and special features as columns in show vendors
2. makes show vendors table horizontally scrollable
3. fixes count values for special features drop down

Testing, didn't opt in to special features:
Screen Shot 2021-12-03 at 10 18 38 PM
Screen Shot 2021-12-03 at 10 18 51 PM

Here I opt in to special features:
Screen Shot 2021-12-03 at 10 21 15 PM

### CODE REVIEW

File: src/js/vendorUtils.js [hunk 1]
```diff
@@ -19,45 +19,34 @@ function findVendor(id, vendorList) {
   return vendorList.vendors[id];
 }
 
-function getHeaderColumn(textContent) {
```
Comment (katie-ta) [inline src/js/vendorUtils.js:22]: thank you for simplifying!

Comment (katie-ta) [reply src/js/vendorUtils.js:22]: wonder who gave you the idea 😏

Comment (charles-tan) [reply src/js/vendorUtils.js:22]: np, I figured it all out by myself

File: src/popup/ucookie.css [hunk 2]
```diff
@@ -113,7 +114,7 @@ a:visited {
 
 .show_button {
   margin-top: 10px;
-  min-width: 50%;
+  min-width: 100%;
```
Comment (katie-ta) [inline src/popup/ucookie.css:117]: i think we would want `width: 100%` at this point

File: src/popup/ucookie.css [hunk 3]
```diff
@@ -236,7 +234,7 @@ a:visited {
   overflow-y: auto;
   max-height: 230px;
   padding-top: 8px;
-  margin-right: 8px;
+  max-width: 484px; /* width of container is 500px, content-container has 8px padding */
```
Comment (katie-ta) [inline src/popup/ucookie.css:237]: nit: `width of body is 500px...`

**Review by katie-ta — APPROVED** [review]
some nits, but overall LGTM!

**Pull request merged**